### DISCUSSION TITLE
Megamenu using antd

### DISCUSSION THREAD

**Discussion by vitalijalbu** [ant-design/ant-design#43016] [General]
Hi guys, anyone ever build a megamenu using antd components? Such as drawer for responsive...?

**Comment by bombillazo** [comment]
What is a mega menu?

**Reply by vitalijalbu** [reply]
https://www.serverplan.com/blog/wp-content/uploads/2020/09/mega-menu-1024x653.png

**Comment by vitalijalbu** [comment]
Hi, anyone can help? @zombieJ do you know anyone who already did it?

![mega-menu-1024x653](https://github.com/ant-design/ant-design/assets/40239902/fad0396d-15d1-4d05-81c6-e723a7e4d0d6)
